### ISSUE TITLE
Why only 2 sectors have been checked ?

### ISSUE THREAD

**Issue by long568** [filecoin-project/lotus#3899]
Please provide all the information requested here to help us troubleshoot "commit failed" issues.
If the information requested is missing, we will probably have to just ask you to provide it anyway,
before we can help debug.

**Describe the problem**
There are 87 sectors on my machine.
The past few days, only 2 sectors have been checked ? 
```
$lotus-miner actor control list
owner      t015051  t3vj7frno...  other  0.565499411847697592 FIL
worker     t015051  t3vj7frno...  other  0.565499411847697592 FIL
control-0  t019035  t3r47ytey...  post   1.763661744671712157 FIL
```
Should I need more FILs ? But the Faucet seems not workring now.

**Sectors status**
Not only one sector's problem.

**Lotus miner logs**
```
Sep 17 11:56:30 m001 lotus-miner[119077]: 2020-09-17T11:56:30.162Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67913
Sep 17 11:56:30 m001 lotus-miner[119077]: 2020-09-17T11:56:30.505Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67913
Sep 17 11:57:00 m001 lotus-miner[119077]: 2020-09-17T11:57:00.356Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67914
Sep 17 11:57:00 m001 lotus-miner[119077]: 2020-09-17T11:57:00.453Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67914
Sep 17 11:57:00 m001 lotus-miner[119077]: 2020-09-17T11:57:00.507Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67914
Sep 17 11:57:12 m001 lotus-miner[119077]: 2020-09-17T11:57:12.448Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67914
Sep 17 11:57:14 m001 lotus-miner[119077]: 2020-09-17T11:57:14.340Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67914
Sep 17 11:57:30 m001 lotus-miner[119077]: 2020-09-17T11:57:30.270Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67915
Sep 17 11:57:30 m001 lotus-miner[119077]: 2020-09-17T11:57:30.286Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67915
Sep 17 11:57:30 m001 lotus-miner[119077]: 2020-09-17T11:57:30.885Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67915
Sep 17 11:57:34 m001 lotus-miner[119077]: 2020-09-17T11:57:34.262Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67915
Sep 17 11:58:00 m001 lotus-miner[119077]: 2020-09-17T11:58:00.339Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67916
Sep 17 11:58:00 m001 lotus-miner[119077]: 2020-09-17T11:58:00.576Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67916
Sep 17 11:58:00 m001 lotus-miner[119077]: 2020-09-17T11:58:00.652Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67916
Sep 17 11:58:00 m001 lotus-miner[119077]: 2020-09-17T11:58:00.866Z        INFO        storageminer        storage/wdpost_sched.go:199        not starting windowPost yet, waiting for startconfidence67912 67916 67916
Sep 17 11:58:30 m001 lotus-miner[119077]: 2020-09-17T11:58:30.452Z        INFO        storageminer        storage/wdpost_sched.go:209        at 67917, doPost for P 67852, dd 1
Sep 17 11:58:30 m001 lotus-miner[119077]: 2020-09-17T11:58:30.460Z        WARN        storageminer        storage/wdpost_run.go:106        Checked sectors        {"checked": 2, "good": 2}
Sep 17 11:58:30 m001 lotus-miner[119077]: 2020-09-17T11:58:30.492Z        INFO        storageminer        storage/wdpost_run.go:407        running windowPost        {"chain-random": "9SeGz/usawgvIbQpCWm1y7tTty8iYB3meaWpUp7UdsQ=", "deadline": {"CurrentEpoch":67917,"PeriodStart":67852,"Index":1,"Open":67912,"Close":67972,"Challenge":67892,"FaultCutoff":67842,"WPoStPeriodDeadlines":48,"WPoStProvingPeriod":2880,"WPoStChallengeWindow":60,"WPoStChallengeLookback":20,"FaultDeclarationCutoff":70}, "height": "67917", "skipped": 0}
Sep 17 12:04:40 m001 lotus-miner[119077]: 2020-09-17T12:04:40.306Z        INFO        storageminer        storage/wdpost_run.go:424        computing window PoSt        {"elapsed": 369.813468638}
Sep 17 12:04:40 m001 lotus-miner[119077]: 2020-09-17T12:04:40.306Z        INFO        storageminer        storage/wdpost_run.go:456        submitting window PoSt
Sep 17 12:04:41 m001 lotus-miner[119077]: 2020-09-17T12:04:41.084Z        INFO        storageminer        storage/wdpost_run.go:526        Submitted window post: bafy2bzacebst32bdfn5qxaoon2yoma77tuvegr6ttz3h3irsndfzhtxzvwxvu
```

**Lotus miner diagnostic info**
http://www.mados.net/res/log/allinfo_20200917

** Code modifications **
None

**Version**
lotus version 0.7.0+git.b89aa02d.dirty

**Comment by jennijuju** [comment]
You lotus version doesn’t look like you did do any code modification

**Comment by long568** [comment]
I download the code by running "git clone https://github.com/filecoin-project/lotus.git lotus". I did not specify the version.

**Comment by chengmou** [comment]
We have the same problem. The official program is rubbish

root@cheng51:~# lotus-miner actor control list
name    ID      key           use         balance                    
owner   t09722  t3v7ulrny...  other post  29.885577736720713062 FIL  
worker  t09722  t3v7ulrny...  other post  29.885577736720713062 FIL

**Comment by jennijuju** [comment]
@long568 have you resolved this issue after updating to 0.7.1? and by `only 2 sectors are c checked`, do you mean only 2 sectors are being verified for every 48 window?

**Comment by long568** [comment]
> @long568 have you resolved this issue after updating to 0.7.1? and by `only 2 sectors are c checked`, do you mean only 2 sectors are being verified for every 48 window?

Now, the issue seems gone. I need more testing to confirm.
